app/routes/bdb/components/BdbDetail.tsx

```diff
@@ -87,6 +102,7 @@ const BdbDetail = () => {
             queryString: queryString(companyId),
           }),
         ),
+        dispatch(fetchAllJoblistings({ company: companyId })),
```

Comment:
If you du `timeFilter=true` by default in backend, I guess we will need `timeFilter=false` here